WooCommerce/WooCommerceUITests/Tests/OrdersTests.swift

```diff
@@ -34,11 +34,15 @@ final class OrdersTests: XCTestCase {
             .addProduct(byName: products[0].name)
             .addCustomerDetails(name: "Mira")
             .createOrder()
+
+        XCTAssertTrue(try SingleOrderScreen().isLoaded)
```

Comment:
for consistency with other tests, wdyt of moving the asserts to the "screen" level and not on the test. e.g. here it could be something like `verifySingleOrderScreenIsLoaded()`?

Reply:
Sure, it's better for consistency. Just included it in b9427cad292412cbc30c5ffac661a97464938a25. Thanks!